src/Models/User2FaCode.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
```

Comment:
```suggestion
```

Remove this import to use our custom Model class with the CamelCaseAttributes trait.

Reply:
fixed